Detectors/DCS/testWorkflow/DCSRandomDataGeneratorSpec.h

```diff
@@ -56,8 +56,14 @@ using HintType = std::variant<DataPointHint<double>,
 {
   std::uniform_int_distribution<int> distribution(min, max);
   std::mt19937 generator;
-  std::vector<int> data(size);
-  std::generate(data.begin(), data.end(), [&]() { return distribution(generator); });
+  std::vector<int> data;
```

Comment:
@aphecetche , this is to make the generation of the DPs in the delta map such that there are no duplicated DPs - the delta map will contain one value only per DP.